### ISSUE TITLE
Mobile-first UI

### ISSUE THREAD

**Issue by Floppy** [openpolitics/votebot#32]
We need a UI redesign so that the app behaves nicely on mobile screens and then scales up. Currently it's a bit of a mess.

**Comment by philipjohn** [comment]
Just noticed an example - portrait tablet (Android, Chrome, Pixel C)

![screenshot_20170301-215335](https://cloud.githubusercontent.com/assets/136342/23483232/fece4bea-fec9-11e6-843d-d9e431ae779f.png)

**Comment by Floppy** [comment]
Yeah, I noticed that page was badly broken on my phone earlier, too. No comments at all for some reason. *When* will I learn to test on mobile as part of the main development??

**Comment by philipjohn** [comment]
When an evil person takes away your laptop and forces you to code everything on a Nokia 3310? :D
